src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -1342,7 +1342,9 @@ private void LoadMembers()
                 if (IsTupleType)
                 {
                     int originalCount = members.Count;
-                    members = AddOrWrapTupleMembers(members.ToImmutableAndFree());
+                    var peMembers = members.ToImmutableAndFree();
+                    members = MakeSynthesizedTupleMembers(peMembers);
+                    members.AddRange(peMembers);
                     membersCount += (members.Count - originalCount); // account for added tuple error fields
```

Comment:
>membersCount += (members.Count - originalCount); / [](http://example.com/codeflow?start=20&length=50)

It feels like this task can be simplified now. We just need to add amount of items returned by ```MakeSynthesizedTupleMembers``` top the current ```membersCount```.